package.json

```diff
@@ -4,16 +4,19 @@
 	"private": true,
 	"scripts": {
 		"start": "node node_modules/react-native/local-cli/cli.js start",
-		"test": "jest"
+		"test": "jest",
+		"postinstall": "rm node_modules/redux-undo/.babelrc"
```

Comment:
This post-install step is a hack that keeps react-native from choking on the `redux-undo` module. Something about React Native not handling Babel plugins from sub-projects.